content/src/controller/Controller.ts

```diff
@@ -525,8 +527,12 @@ export class Controller {
         nextFilters.to = lastDeployment.entityTimestamp
       }
     }
+
+    const fields = !options.fields || options.fields === DEFAULT_FIELDS_ON_DEPLOYMENTS ? '' : options.fields.join(',')
+
     const nextQueryParams = toQueryParams({
       ...nextFilters,
+      fields,
```

Comment:
Is there a way to be sure that in the future we don't forget to add anything?

Reply:
Not that I can think of right now